src/funtracks/actions/add_delete_node.py

```diff
@@ -79,6 +79,12 @@ def _apply(self) -> None:
         for attr, values in attrs.items():
             self.tracks._set_node_attr(self.node, attr, values)
 
+        # track_id is required for SolutionTracks
```

Comment:
TODO for later: put all validation in the inits, and update the docstrings with the exceptions, for all the actions